scripts/build-cli-bundles.ts

```diff
@@ -1,32 +1,109 @@
 /// <reference types="@types/bun" />
 
-import { rm } from 'node:fs/promises';
+/*
+This script is responsible for creating the CLI bundles (one file that embeds runtime with the CLI code, that can be just downloaded and ran).
+
+At the time of writing (~2025-05-31), the bundles are created using Bun as the runtime and the targets you can see in the code.
+When node stabilizes SEA (https://nodejs.org/api/single-executable-applications.html) [and supports ESM -.-], this code can be adapted to build it using that instead (but cross-platform will be a CI experience)
+*/
+
+import { readFile, rm, writeFile } from 'node:fs/promises';
 import { basename } from 'node:path';
 
 import { $, fileURLToPath } from 'bun';
 
 import { version } from '../package.json' with { type: 'json' };
 
-const targets = ['bun-linux-x64', 'bun-linux-arm64', 'bun-windows-x64', 'bun-darwin-arm64', 'bun-darwin-x64'];
+const targets = (() => {
+	if (process.env.APIFY_FULL_CLI_BUNDLES) {
+		return [
+			//
+			'bun-windows-x64',
+			'bun-windows-x64-baseline',
+			'bun-linux-x64',
+			'bun-linux-x64-baseline',
+			'bun-linux-arm64',
+			'bun-linux-arm64-baseline',
+			'bun-darwin-x64',
+			'bun-darwin-x64-baseline',
+			'bun-darwin-arm64',
+			'bun-darwin-arm64-baseline',
+			'bun-linux-x64-musl',
+			'bun-linux-arm64-musl',
+			'bun-linux-x64-musl-baseline',
+			'bun-linux-arm64-musl-baseline',
+		];
+	}
+
+	if (process.platform === 'win32') {
+		return ['bun-windows-x64', 'bun-windows-x64-baseline'];
+	}
+
+	return [
+		'bun-linux-x64',
+		'bun-linux-x64-baseline',
+		'bun-linux-arm64',
+		'bun-linux-arm64-baseline',
+		'bun-darwin-x64',
+		'bun-darwin-x64-baseline',
+		'bun-darwin-arm64',
+		'bun-darwin-arm64-baseline',
+		'bun-linux-x64-musl',
+		'bun-linux-arm64-musl',
+		'bun-linux-x64-musl-baseline',
+		'bun-linux-arm64-musl-baseline',
+	];
+})();
 
 const entryPoints = [
 	//
 	fileURLToPath(new URL('../src/entrypoints/apify.ts', import.meta.url)),
+	fileURLToPath(new URL('../src/entrypoints/actor.ts', import.meta.url)),
 ];
 
 await rm(new URL('../bundles/', import.meta.url), { recursive: true, force: true });
 
+// #region Inject the fact the CLI is ran in a bundle, instead of installed through npm/volta
```

Comment:
how does the `#region` comment work? is that something bun specific? first time seeing that i guess